import java.util.Hashtable;
import java.util.Properties;

import javax.naming.Context;
import javax.naming.InitialContext;
import javax.naming.NamingException;

import business_objects.User;


public class test {

	/**
	 * @param args
	 */
	public static void main(String[] args) {

		User user = new User("Martin", "Ecoffey", "m.ecoffey@gmail.com");
		
		try {
			Hashtable env = new Hashtable();
			env.put(Context.PROVIDER_URL, "ldap://ldap.wiz.com:389");//mysql://localhost:3306
			env.put(Context.SECURITY_PRINCIPAL, "root");
			env.put(Context.SECURITY_CREDENTIALS, "");
			
			Properties props = new Properties();
		      props.put(Context.INITIAL_CONTEXT_FACTORY,
		          "com.sun.jndi.fscontext.RefFSContextFactory");
		      props.put(Context.PROVIDER_URL, "file:///");
			
			Context ctx = new InitialContext(props);
			
			NutritionService service = (NutritionService)ctx.lookup("NutritionService/local");
			service.storeUser(user);
			
		} catch (NamingException e) {
			e.printStackTrace();
		}
	}
}
